src/LtiGradeSubmissionReview.php

```diff
@@ -41,7 +41,7 @@ public function getReviewableStatus()
         return $this->reviewable_status;
     }
 
-    public function setReviewableStatus($value)
+    public function setReviewableStatus($value): self
```

Comment:
Likewise, the parameters and class attributes referenced in these functions aren't typed.